extensions/wikia/AnalyticsEngine/AnalyticsProviderIVW3.php

```diff
@@ -45,8 +60,78 @@ private function trackPageView() {
 				'countries' => json_encode( $wgAnalyticsDriverIVW3Countries ),
 				'siteId' => self::$siteId,
 				'url' => self::$libraryUrl,
-				'vertical' => HubService::getVerticalNameForComscore( $wgCityId ),
+				'cmKey' => self::getCMKey()
 			]
 		);
 	}
+
+	static public function getCMKey() {
+
+		$language = self::getMappedLanguage();
+		$pageType = self::getPageType();
+		$theme = self::getTheme();
+		$vertical = self::getVertical();
```

Comment:
How come? Would it be less readable without the vars? You can write it without them in a readable form IMHO :smile_cat: